components/Record/utils/postPost.ts

```diff
@@ -1,26 +1,20 @@
-import { APIKit, getToken } from '../../../shared/APIkit';
+import React from 'react';
+import { APIKit, getToken, onFailure } from '../../../shared/APIkit';
 import Constants from 'expo-constants';
 import * as FileSystem from 'expo-file-system';
-
 const { manifest } = Constants;
 
-async function postSound(
+async function PostPost(
```

Comment:
Är PostPost sånt bra namn?

Reply:
The struggle av att vi kallar det "post" istället för "Goj" 😅 . Förslag att byta namnet till `createPost`, vad tycks?

Reply:
Mycket bättre!